src/test/java/de/rabea/player/ComputerPlayerSpeedTest.java

```diff
@@ -0,0 +1,38 @@
+package de.rabea.player;
+
+import de.rabea.game.Board;
+import org.junit.Test;
+
+import static de.rabea.game.Mark.X;
+import static org.junit.Assert.assertTrue;
+
+public class ComputerPlayerSpeedTest {
+
+    @Test(timeout=3200)
+    public void measureSpeedOfFirstMoveOn3x3Board() {
+        UnbeatableComputerPlayer computer = new UnbeatableComputerPlayer(X);
+        Board board;
+        long startTime = System.nanoTime();
+        for (int i = 0; i <= 10; i++) {
+            board = new Board(3);
+            computer.getPosition(board);
+        }
+        long endTime = System.nanoTime();
+        long duration = (endTime - startTime)/1000000;
+        System.out.println("duration in milliseconds= " + duration);
```

Comment:
Not sure about leaving these println statements in these tests. At the very least, when you run all the slow tests, there's no context about what each of the durations are for.